src/Microsoft.Framework.PackageManager/Bundle/BundleOperations.cs

```diff
@@ -17,15 +18,40 @@ public void Delete(string folderPath)
             FileOperationUtils.DeleteFolder(folderPath);
         }
 
-        public void Copy(string sourcePath, string targetPath)
+        public void Copy(IEnumerable<string> sourceFiles, string sourceDirectory, string targetDirectory)
         {
-            Copy(
-                sourcePath, 
-                targetPath, 
-                shouldInclude: _ => true);
-        }
+            sourceDirectory = PathUtility.EnsureTrailingSlash(sourceDirectory);
+            targetDirectory = PathUtility.EnsureTrailingSlash(targetDirectory);
+
+            foreach (var sourceFilePath in sourceFiles)
+            {
+                var fileName = Path.GetFileName(sourceFilePath);
+                Debug.Assert(fileName != null, "fileName != null");
```

Comment:
In that case, IsNullOrEmpty wouldn't change the behavior, right? I think Wei's suggestion makes sense.  In case Matcher changes in the future, that would keep this code path correct.  Alternatively, you can consider adding a test case to cover this.